### ISSUE TITLE
Idea: Add a section about i18n

### ISSUE THREAD

**Issue by l3m** [Zaid-Ajaj/the-elmish-book#81]
We're currently using a homegrown approach for i18n in our Elmish application.

I would be interested in best practices for the support of multiple languages.

**Comment by Zaid-Ajaj** [comment]
Absolutely! This is a really important topic as well. I think it would be an appropriate section for chapter 4: Composing Larger Applications.
